meta/inspiredminds/contao-isotope-edit-order-addresses/de.yml

```diff
@@ -0,0 +1,4 @@
+de:
+    title: Isotope Edit Order Addresses
```

Comment:
sollte der Titel nicht auch übersetzt sein?

Reply:
No, I always use the English name.

Reply:
any specific reason? That means the german package list gets english package names, why would that make sense?

Reply:
Because it is a name. Windows is still called Windows in German and not "Fenster", etc. I want an extension to be recognised (as the same) no matter the language. It does not make sense to me to translate the names of extensions.